exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,10 @@
 // Declare your function here
+const string = "Hello, my name is Daniel and I'm ";
+const stringtwo = " years old";
 
-const greeting = createLongGreeting("Daniel", 30);
+function introduction(age){
+   return string + age + stringtwo;
+}
+const greeting = introduction(30);
 
 console.log(greeting);
```

Comment:
Tersia, your const greeting contains two functions, one is createLongGreeting() and the other one is Introduction() so I wonder what would happen when invoking 2 functions under one variable, please review this situation.